Gracie/dashboard

artsygracie17 opened the pull request.

Commit:
API client framework working and getCurrentUser working. Integrate Google sign in with added View Model; take things out from App Delegate

API client framework working and getCurrentUser working. Integrate Google sign in with added View Model; take things out from App Delegate

Add project pbx

Request, LucienRequest, APIClient, LucienAPIClient framework

Add getCurrentUser endpoint; working

Commit:
Create Dashboard and Dashboard navigation controller. Link profile to navigation smiley icon in dashboard. Decode token in separate AuthenticationResponse model.

Add setNavBarBackground and setNavBarTitle to LucienTheme to be accessed and customized throughout application whenever nav bar needs to be created

Restructure UINavigationController. Navigation flow from Dashboard to Profile working

Authorization token now set properly. Authenticate user everytime at sign in even if already signed in

Commit changes; in middle of getting getCurrentUser endpoint working

getCurrentUser working. Add AuthenticationResponse model for authenticate user and use User model for getCurrentUser only.

Add sample image

Commit:
Begin overall view layout of Dashboard. Add three views for lending borrowing and my collection (now empty). Horizontal scroll view working but nothing in it

Commit changes to checkout. Start UICollectionView

Change collection count decoder coding key to int so that Dashboard gets decoded properly

Grab Dashboard data successfully! So far only decoding myCollectionCount. Add logic to sign in RootViewController to bring user to either start a new collection or to the dashboard depending on myCollectionCount

Commit:
Update StartMyCollection design, components and constraints to match new Zeplin design; include linking to Dashboard

Update StartMyCollection xib design to Zeplin state

Add UILabel extension for letter spacing; StartMyCollection welcome screen UI elements updated

Edit constraints in home screen for aspect ratios conformity on iPhone touch

Tweak StartMyCollection screen for device demo

Commit:
Substitute Dashboard endpoint to has_collection so that the check of whether user has comics in collection or not runs faster

hasCollection with String intrepretation working (was in bytes earlier)

Commit:
Fix transition from welcome screen to AddCommic screen after rebase. Glitches fixed; add hard-coded scrolling on Dashboard xib for demo

Change color of hard coded scrolling views and direction lock

Delete GoogleAuthentication.swift

Revert back Guled AddComic changes; still not working

AddComicController working

Lucien Profile title glitch fixed

Committing changes to rebase on develop for Jenkins stuff

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/09ea6aa691ee3b748ef8f0ec321de884aa387559/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  0 Errors
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/09ea6aa691ee3b748ef8f0ec321de884aa387559/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/09ea6aa691ee3b748ef8f0ec321de884aa387559/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.359 (0.362) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 7.321 (7.323) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/17bf0cbc400c27aa26f319f5d236ba4a81080ba7/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  0 Errors
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/17bf0cbc400c27aa26f319f5d236ba4a81080ba7/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/17bf0cbc400c27aa26f319f5d236ba4a81080ba7/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.384 (0.386) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 6.189 (6.190) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/2e232c9ec47382a5073f93f0eaee67ceef7d9cea/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/2e232c9ec47382a5073f93f0eaee67ceef7d9cea/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/2e232c9ec47382a5073f93f0eaee67ceef7d9cea/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.372 (0.375) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 8.977 (8.979) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/e195e593b977254280b9f20ac782e6c1b3d8a132/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  0 Errors
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/e195e593b977254280b9f20ac782e6c1b3d8a132/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/e195e593b977254280b9f20ac782e6c1b3d8a132/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.378 (0.381) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 16.447 (16.450) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/4ab0d60aa3f11bce721995ecd4fdf6daf3470d81/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4ab0d60aa3f11bce721995ecd4fdf6daf3470d81/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4ab0d60aa3f11bce721995ecd4fdf6daf3470d81/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.380 (0.382) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 6.637 (6.639) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented on lucien-ios/Dashboard/LendingViewController.swift:
```diff
@@ -0,0 +1,49 @@
+        super.viewDidLoad()
+    }
+
+    var myArray = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]
```
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comma Spacing Violation: There should be no space before and one after any comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acomma)

IntrepidBuildServer commented:
SonarQube analysis reported 18 issues
* ![MINOR][MINOR] 18 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/530490001003160a74de2e2f47f0186980cebf66/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/530490001003160a74de2e2f47f0186980cebf66/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/530490001003160a74de2e2f47f0186980cebf66/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.392 (0.394) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 6.749 (6.752) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/530490001003160a74de2e2f47f0186980cebf66/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/530490001003160a74de2e2f47f0186980cebf66/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.403 (0.405) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 25.527 (25.530) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

Commit:
UICollectionView in separate xib file added. Add separate xib for UICollectionViewCell then load xib to main UICollectionView view controller

Change size of LendingCollectionViewCell but not showing up properly. Last commit before breaking PR/branch to smaller parts

Trim extra lines

Remove GoogleAuthentication.swift

Update fonts where they were not updated previously. Change app delegate root controller back to RootViewController

Remove extra whitespace

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/4b52a5b021f41dc77a40f6deaa4210754021da18/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4b52a5b021f41dc77a40f6deaa4210754021da18/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4b52a5b021f41dc77a40f6deaa4210754021da18/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.393 (0.396) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 12.905 (12.908) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/d1233110306ccf65829e539a2e058c99c867a724/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/d1233110306ccf65829e539a2e058c99c867a724/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/d1233110306ccf65829e539a2e058c99c867a724/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.366 (0.369) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 7.798 (7.801) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GoogleAuthentication.swift#L1](https://github.com/IntrepidPursuits/lucien-ios/blob/925bb6211429276bbf6c9d0c8409f6fd9c869693/lucien-ios/APIClient/GoogleAuthentication.swift#L1): Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  0 Errors
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/925bb6211429276bbf6c9d0c8409f6fd9c869693/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/925bb6211429276bbf6c9d0c8409f6fd9c869693/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.376 (0.377) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 7.291 (7.293) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4dc0ba63174a6437519571b6675ebe1ecc42187b/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/4dc0ba63174a6437519571b6675ebe1ecc42187b/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.345 (0.347) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 7.633 (7.635) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented on lucien-ios/ProfileViewController.swift:
```diff
@@ -16,18 +16,61 @@ final class ProfileViewController: UIViewController {
+
+
+
+
```
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)

ghost commented:
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/7e3ede8bfd730d1d4aae4dd6e92ee9c0f42af1b7/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/7e3ede8bfd730d1d4aae4dd6e92ee9c0f42af1b7/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.330 (0.332) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 11.888 (11.890) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/edcc70f95f69daaa4e4ae4e5349257409f8b5dad/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/edcc70f95f69daaa4e4ae4e5349257409f8b5dad/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.417 (0.420) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 8.274 (8.277) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/edcc70f95f69daaa4e4ae4e5349257409f8b5dad/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/edcc70f95f69daaa4e4ae4e5349257409f8b5dad/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.351 (0.353) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 11.496 (11.497) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/d2df9008217bfc2396589945e739b8627e9c80a8/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/d2df9008217bfc2396589945e739b8627e9c80a8/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.320 (0.322) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 6.232 (6.234) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

IntrepidBuildServer commented on lucien-ios/ProfileViewController.swift:
```diff
@@ -16,18 +16,58 @@ final class ProfileViewController: UIViewController {
+        name.font = LucienTheme.Fonts.muliRegular(size: 18.0)
+        email.font = LucienTheme.Fonts.muliRegular(size: 18.0)
+    }
+    
```
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)

IntrepidBuildServer commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/dbaf92ad77402d5c18d5db97000a3309873ed631/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/dbaf92ad77402d5c18d5db97000a3309873ed631/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.350 (0.352) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 6.644 (6.646) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

ghost commented:
<!--
  1 Error: fatal error: file ‘/Users/jenk...
  2 Warnings: Please provide a summary in th...
  2 Messages: Executed 2 tests, with 0 failu...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">fatal error: file ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/Pods/Google/Headers/module.modulemap’ has been modified since the precompiled header ‘/Users/jenkins/Jenkins/workspace/s-pipeline_lucien-ios_PR-11-HPQ6GO3RXE676ILJ37MSNIG65EO52WXH4IOQGW3EQAVY5VEGP6GQ/DerivedData/Build/Intermediates.noindex/PrecompiledHeaders/lucien-ios-Bridging-Header-swift_18QXC9KXB9ZK2-clang_2O9YIEWVALUCS.pch’ was built</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/dbaf92ad77402d5c18d5db97000a3309873ed631/CameraViewController.swift#L81">CameraViewController.swift#L81</a> - <strong><a href="https://github.com/IntrepidPursuits/lucien-ios/blob/dbaf92ad77402d5c18d5db97000a3309873ed631/lucien-ios/Camera/CameraViewController.swift#L81">lucien-ios/Camera/CameraViewController.swift#L81</a></strong>: value ‘image’ was defined but never used; consider replacing with boolean test  <br /><code>
            if let image = getImageFromSampleBuffer(buffer: sampleBuffer) {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 2 tests, with 0 failures (0 unexpected) in 0.326 (0.329) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 1 test, with 0 failures (0 unexpected) in 9.333 (9.336) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>
